websites/B/BreathFM/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "846282107462352927",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData;
+	if ("mediaSession" in navigator && navigator.mediaSession.metadata !== null) {
```

Comment:
Otherwise it won't show the other parts of the site when it's paused.
```suggestion
	if ("mediaSession" in navigator && navigator.mediaSession.metadata !== null && !document.querySelector("#root > audio")?.paused) {
```